### PR TITLE
Update entryTypeDefinition description

### PR DESCRIPTION
**Pull request by mbaudis**
The parameter descriptions in the EntryTypeDefinitions schema seem somewhat hard to parse and potentially misleading - this is a go at cleaning them up a bit.

### CODE REVIEW

**Comment by mbaudis** [comment]
It is not clear how to use ontologyTermForThisType w/o examples (i.e. terms for biosample, individual... - added some but incomplete...) and there is also a logical duplication having a "unique" id and an ontology term. Overall this should be clarified but the current modification seems (a) sensible (step).

**Review by costero-e — APPROVED** [review]
I like the changes you made, as it makes it more clear to know what is an entry Type. Thank you. I approved the changes but I have a particular opinion (that maybe is wrong) for the entry Type description: **Definition of an element or entry type, to describe each type of entry type included in a beacon**
I think it would be better to not use "entry Type" in its own definition, maybe we could use "scope" here, as I see it as, correct me if I'm wrong, a similar term for entry Type. For example we could have something like: **Definition of an element or scope of the element, to describe each type of entry type included in a beacon**
The rest I see it correct and you are right we should extend the list of _ontologyTermForThisType_.

**Comment by mbaudis** [comment]
> Definition of an element or scope of the element, to describe each type of entry type included in a beacon

Excellent!

**Comment by costero-e** [comment]
> > Definition of an element or scope of the element, to describe each type of entry type included in a beacon
> 
> Excellent!

Updated the definition with a new commit!

**Comment by jrambla** [comment]
The purpose of _ontologyTermForThisType_ is to allow a semantic description of the model.
A new model (e.g. on cancer registries) with a whole set of entryTypes, or a new entryType (e.g. File or a genomicFeature) would have this attribute to describe which type of thing that endpoint is.
It is NOT an identifier, but a mechanism for make the models shallowly semantic.

**Review by jrambla — APPROVED** [review]
